packages/rich-text-editor/docs/RichTextEditor.stories.tsx

```diff
@@ -19,11 +21,24 @@ export default {
 export const Default = args => {
   const [rteData, setRTEData] = useState<EditorContentArray>([])
   return (
-    <RichTextEditor
-      value={rteData}
-      onChange={data => setRTEData(data)}
-      {...args}
-    />
+    <>
+      <RichTextEditor
+        value={rteData}
+        onChange={data => setRTEData(data)}
+        {...args}
+      />
+      <Box pt={1}>
+        <InlineNotification
+          type="cautionary"
+          title="Security Hint"
+          hideCloseIcon
+        >
+          Our Rich Text Editor deals with user input which can potentially
+          include malicious content (e.g. XSS, injection attacks). It should
+          always be treated with great care when processing or displaying again.
+        </InlineNotification>
+      </Box>
```

Comment:
Should this be part of the story? Concerned some may take this to mean they need to include it when they use the RTE for build.

Reply:
You mean like those birthday cakes where the instruction is "Please write something funny on it" and they literally write "Please write something funny on it" on the cake? ;)
I'm not too worried about this, also can't really think of a better place in our current docs.